### DISCUSSION TITLE
part-ii/2018-114/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#46] [Question comments]
Part II 2018 Principles of Quantum Mechanics: Paper 3, Section II, D 

https://questions.tripos.org/part-ii/2018-114/

**Comment by silkca** [comment]
The first part is demonstrated in lectures.

It should be noted for the second part that the situation does not obey Maxwell's equations (recent exposure to AQM led me astray at this point), and the contribution to the Hamiltonian by the electric field is just $\Delta(t)=|\mathbf{z}|\mathbf{E}(t)$.

Performing the integrals involved is certainly tedious, but not particularly mathematically stimulating. Finding the proportion of atoms in the $|2,0,0\rangle$ state after a long time requires thought about symmetries, not calculation.

**Reply by thenewtonium** [reply]
Isn't the contribution to the hamiltonian by the electric field $\mathbf{X}\cdot\mathbf{E}(t)$?

Anyway, for those watching at home...
<details><summary>Elaborating on the symmetry</summary>

The key point is that the wavefunctions of both $|2,0,0\rangle$ and $|1,0,0\rangle$ are even functions of $z$, hence $\langle 2,0,0|Z|1,0,0 \rangle$ vanishes. You should notice this fairly quickly anyway if you do just try to calculate $\langle 2,0,0|Z|1,0,0 \rangle$ directly though (as I did oops), since the polar ($\theta$) dependence integrates to $0$...
</details>
